src/dotnet-scaffolding/dotnet-scaffold-aspnet/Extensions/ViewScaffoldeBuilderExtensions.cs

```diff
@@ -0,0 +1,63 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+using Microsoft.DotNet.Scaffolding.Core.Builder;
+using Microsoft.DotNet.Scaffolding.TextTemplating;
+using Microsoft.DotNet.Tools.Scaffold.AspNet.Helpers;
+using Microsoft.DotNet.Tools.Scaffold.AspNet.Models;
+using Microsoft.DotNet.Tools.Scaffold.AspNet.ScaffoldSteps;
+using Microsoft.DotNet.Tools.Scaffold.AspNet.ScaffoldSteps.Settings;
+
+namespace Microsoft.DotNet.Scaffolding.Core.Hosting;
+
+internal static class ViewScaffoldeBuilderExtensions
+{
+    public static IScaffoldBuilder WithViewsTextTemplatingStep(this IScaffoldBuilder builder)
+    {
+        return builder.WithStep<TextTemplatingStep>(config =>
+        {
+            var step = config.Step;
+            var context = config.Context;
+            context.Properties.TryGetValue(nameof(ViewModel), out var viewModelObj);
+            ViewModel viewModel = viewModelObj as ViewModel ??
+                throw new InvalidOperationException("missing 'ViewModel' in 'ScaffolderContext.Properties'");
```

Comment:
Same comment as in `EfControllerScaffolderBuilderExtensions.cs`

Reply:
yep fixed it as well